src/linear-client.ts

```diff
@@ -82,9 +82,10 @@ export class LinearClient extends SharedEndpoints {
     stop_loss?: number;
     tp_trigger_by?: string;
     sl_trigger_by?: string;
-    reduce_only?: boolean;
-    close_on_trigger?: boolean;
+    reduce_only: boolean;
+    close_on_trigger: boolean;
     order_link_id?: string;
+    position_idx: number;
```

Comment:
Thanks for the contribution! Looks like this last param is optional (unless you're in one-way mode):
```suggestion
    position_idx?: number;
```

https://bybit-exchange.github.io/docs/linear/#t-placeactive